notebookUtils.py

```diff
@@ -45,7 +45,7 @@ def saveFile(file_path, token, log):
     else:
         url = 'http://data-loader-api/data-loader-api/save'
     headers = {'X-StorageApi-Token': token, 'User-Agent': 'Keboola Sandbox Autosave Request'}
-    payload = {'file': {'source': file_path, 'tags': ['autosave']}}
+    payload = {'file': {'source': file_path, 'tags': ['autosave', 'sandbox' + os.environ['SANDBOX_ID']]}}
```

Comment:
I guess it doesn't really matter since it's constant, but maybe it'd look better with a `-`?

Reply:
Aha, I thought that it is there, a mistake probably. ;o)